sig-big-data/charter.md

```diff
@@ -0,0 +1,56 @@
+# SIG Big Data Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+The Big Data SIG serves as a community resource for advising big data and data science related software projects on techniques and best practices for integrating with Kubernetes. This SIG also represents the concerns of users from big data communities to Kubernetes for the purposes of driving new features and other enhancements, based on big data use cases.
+
+### In scope
+
+#### Code, Binaries and Services
```

Comment:
Are there OWNERS files maintained by the SIG in the Kubernetes org?

Reply:
currently, no